### PR TITLE
Translate i18n/zh/docusaurus-plugin-content-docs/current/developers/references/component-types/worker.md

### PR DESCRIPTION
**Pull request by chenggui53**
…eferences/component-types/worker.md

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/developers/references/component-types/worker.md [hunk 1]
```diff
@@ -20,9 +20,9 @@ services:
...
 
-## Properties
+## 属性
 
-Name | Description | Type | Required | Default 
+名称 | 描述 | 类型 | 是否必须 | 默认值 
 ------------ | ------------- | ------------- | ------------- | ------------- 
- cmd | Commands to run in the container | []string | false |  
- image | Which image would you like to use for your service | string | true |  
+ cmd | 容器中运行的命令 | []string | 否 |
```
Comment (sunny0826) [inline i18n/zh/docusaurus-plugin-content-docs/current/developers/references/component-types/worker.md]: `true` 和 `false` 不需要翻译

**Review by sunny0826 — CHANGES_REQUESTED** [review]
稍加翻译即可

**Review by sunny0826 — APPROVED** [review]
lgtm

File: i18n/zh/docusaurus-plugin-content-docs/current/developers/references/component-types/worker.md (hunk 1 above)
Comment (chenggui53) [reply i18n/zh/docusaurus-plugin-content-docs/current/developers/references/component-types/worker.md]: 已更新

**Pull request merged**